_data/footer.yml

```diff
@@ -1,4 +1,4 @@
-copyright: 'This site uses Cookies. We use cookies to help the website run effectively. To find out more, read our [Privacy](https://elixir-europe.org/privacy) and [Cookies](https://elixir-europe.org/cookies) statements.'
```

Comment:
It was decided that the privacy and cookie notices should be removed from the footer. It was also decided to add in a copyright notice. My suggestion is to skip adding in the years. If that would be desired, the implementation should be automatic, i.e., not requiring the years to be edited manually.